packages/ui/src/utils/utilsNetworks.ts

```diff
@@ -288,20 +291,21 @@ export const baseNetworksConfig = {
     scanTokenPath: (hash: string) => `https://fraxscan.com/token/${hash}`,
   },
   196: {
```

Comment:
It would be nice to create an enum directly in this file with all ChainId's and then use it instead of the numbers as is.
```ts
enum ChainId {
  MAINNET: 1,
  ...
  X_LAYER: 196
}
```

```suggestion
  ChainId.X_LAYER: {
```